src/SFA.DAS.FAT.Web/Models/CourseViewModel.cs

```diff
@@ -26,11 +28,15 @@ public class CourseViewModel : PageLinksViewModelBase
     public int TypicalDuration { get; set; }
     public string TypicalJobTitles { get; set; }
     public string StandardPageUrl { get; set; }
-    public string[] Skills { get; set; }
-    public string[] Knowledge { get; set; }
-    public string[] Behaviours { get; set; }
+    public ApprenticeshipType ApprenticeshipType { get; set; }
+    public bool IsFoundationApprenticeship => ApprenticeshipType == ApprenticeshipType.FoundationApprenticeship;
```

Comment:
Since this is used a few times in the view, we can set this in the operator once instead of calculating each time.